src/components/NameBox.js

```diff
@@ -0,0 +1,31 @@
+import React, {useState , useEffect} from 'react'
+
+export const NameBox = () => {
+  const [name, setName] = useState("");
+  useEffect(() => {
+    
+  }, []);
+
+  const onChange = (e) => {
+    console.log(name);
```

Comment:
At some point we'll need to take this out for cleanliness

Reply:
yup